frontend/src/components/RegistrationView/Form.tsx

```diff
@@ -0,0 +1,224 @@
+                            error={errors.juniorGender} 
+                            touched={touched.juniorGender} 
+                            title="Sukupuoli" 
+                            name="juniorGender" 
+                            options={['Tyttö', 'Poika', 'Muu', 'En halua määritellä']}
+                        />
+
+                        <SelectGroup 
+                            error={errors.kuvauslupa} 
+                            touched={touched.kuvauslupa} 
+                            title="Kuvauslupa" 
+                            name="kuvauslupa" 
+                            description="Valokuvaamme ja videoimme ajoittain toimintaamme ja nuoria viestintää varten. Kuvia voidaan käyttää Nuorisopalveluiden jalkaisuissa (esim. sosiaalisessa mediassa, nettisivuilla ja esitteissä). \nLapseni kuvaa saa käyttää lapsen asuinkaupungin viestinnässä."
+                            options={['Kyllä', 'Ei']}/>
+
+                    </Fieldset>
+                </Column>
+
+                <Column>
+                    <Fieldset>
+                        <FieldTitle>Huoltajan tiedot</FieldTitle>
+                        <Field name='parentFirstName' component={InputField} title='Etunimi'/>
+                        <Field name='parentLastName' component={InputField} title='Sukunimi'/>
+                        <Field name='parentPhoneNumber' component={InputField} title='Puhelinnumero'/>
+                    </Fieldset>
+
+                    <Fieldset>
+                        <FieldTitle>Kotinuorisotila</FieldTitle>
+                        <Field name='youthClub' 
+                            component={DropdownField} 
+                            title='Kotinuorisotila' 
+                            options={['club1', 'club2', 'club3']}
+                            defaultChoice='Kannu'
+                            description="Valitse nuorisotila, jossa lapsesi tai nuoresi yleensä käy."
+                        />      
+                </Fieldset>
+            </Column>
+            <FormFooter>
+                <Button type="submit">Lähetä hakemus</Button>
+                <a href="#">Lue tarkemmin, kuinka käsitelemme tietojasi.</a>
+            </FormFooter>
+        </Form>
+    )
+}
+
+interface RegFormProps {
+    parentFirstName?: string,
+    parentLastName?: string,
+    onSubmit: () => void,
+}
+
+
+const RegistrationForm = withFormik<RegFormProps, FormValues>({
+    mapPropsToValues: props => {
+        return {
+            juniorFirstName: '',
+            juniorLastName: '',
+            juniorNickName: '',
+            juniorBirthday: '',
+            juniorPhoneNumber: '',
+            juniorGender: '',
+            postalCode: '',
+            kuvauslupa: '',
+            school: '',
+            class: '',
+            parentFirstName: '',
+            parentLastName: '',
+            parentPhoneNumber: '',
+            youthClub: ''
+        }
+    },
+    validationSchema: object().shape({
```

Comment:
Not critical, but these could be stored in some other place than component code. Seperate validation directory would do.